src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/SemanticModelReuse/SemanticModelWorkspaceServiceFactory.SemanticModelWorkspaceService.cs

```diff
@@ -62,8 +62,7 @@ public SemanticModelReuseWorkspaceService(Workspace workspace)
         {
             _workspace = workspace;
 
-#pragma warning disable RS0030 // Do not use banned APIs
-#if WORKSPACE
+#if WORKSPACE || CODE_STYLE
```

Comment:
I'm a little confused by the various directives now, but this builds, which I guess means it's right. @JoeRobich?

Reply:
I eventually figured out ROSLYN_4_12_OR_LOWER is the right condition to use.